app/templates/components/public/sponsor-list.hbs

```diff
@@ -1,9 +1,7 @@
 <h3 class="ui header">{{t 'Sponsors'}}</h3>
-{{#each-in sponsorsGrouped as |key sponsors|}}
```

Comment:
Hi @niranjan94! After been trying to playing around with the sponsor API if I remove this then I am able to display the sponsors on the page but if I keep this then I am getting this error 
![image](https://user-images.githubusercontent.com/12701036/28246264-846ea9c8-6a33-11e7-9b35-387caec600f7.png)

The issue is with the `sponsorGrouped` method in the `sponsor-list.js` file. We're passing this `'sponsors.[]'` currently. I have tried to change this but didn't understand what exactly should come here.
Can you please guide? Thanks.